### ISSUE TITLE
A "select all and remove" button to delete n number of item lines together

### ISSUE THREAD

**Issue by sworup** [openmsupply/mobile#1664]
## Please describe the feature being requested.

- In mobile, it is very difficult to multiple select multiple lines and delete them together. 
- I think we need to add select all trigger, on click, of `Remove` button which when pressed would select all of the items for removal/deletion. 
- We can go the same route of the single delete, when a conformation popup shows up at the button before we can finalise the deletion.

## Describe the solution

![Screenshot_2019-12-10_at_11_20_56](https://user-images.githubusercontent.com/683452/70498762-8ebf1b00-1b3f-11ea-8694-31ec51cbb269.jpg)

### Describe alternatives you've considered

None

### Additional context

None

**Comment by joshxg** [comment]
I like this - just curious, was this a suggestion from a user?

**Comment by sworup** [comment]
> I like this - just curious, was this a suggestion from a user?

@joshxg No, I created this issue based on my own experience, the difficulty I had trying to bulk delete lines.

@anildahalsussol wants to take this issue when we have go ahead to do it.
